pages/pie-chart/test.page.tsx

```diff
@@ -38,6 +39,11 @@ export default function () {
               ariaLabel="Food facts"
               size="medium"
               onHighlightChange={e => setActiveSegment(e.detail.highlightedSegment)}
+              detailPopoverFooter={segment => (
+                <Box margin={{ top: 'm' }}>
+                  <Button>Filter by {segment.title}</Button>
```

Comment:
Why do we have different formats for pie- and other charts? Would it be safer to make the first argument always an object which can be extended with more properties in the future?

Reply:
Conceptually, we always pass the same data, it's just called differently in the components. But this is always the "x" value that is of the generic `T` type. But in pie chart we call them segments.

Reply:
I currently cannot imagine any other data that would be useful to pass down here. So I'd rather not have a one-key object paramter.

Reply:
We can pass formatted value, series, total. Of course, we can always pass those as an extra argument if ever needed.
I don't mind keeping it as is.